test failing with:

   235  unknown location(0): ^[[4;31;49mfatal error: in "test_signers/test_generic_signers": std::runtime_error: Asked to create (C)DS record of unknown digest type 4: Request to create key object fo
r unknown algorithm number 14^[[0;39;49m
   236  test-signers.cc(425): ^[[1;36;49mlast checkpoint^[[0;39;49m
   237  ^[[1;36;49mTest case test_packetcache_cc/test_QueryCacheClean did not check any assertions^[[0;39;49m
   238
   239  ^[[1;31;49m*** 1 failure is detected in the test module "Master Test Suite"
   240  ^[[0;39;49mFAIL testrunner (exit status: 201)

--- a/pdns/test-signers.cc
+++ b/pdns/test-signers.cc
@@ -453,31 +453,6 @@ static void test_generic_signer(std::sha
 }
 
 // NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables,readability-identifier-length): Boost stuff.
-BOOST_FIXTURE_TEST_CASE(test_generic_signers, Fixture)
-{
-  for (const auto& algoSignerPair : signerParams) {
-    auto signer = algoSignerPair.second;
-
-    DNSKEYRecordContent drc;
-    auto dcke = std::shared_ptr<DNSCryptoKeyEngine>(DNSCryptoKeyEngine::makeFromISCString(drc, signer.iscMap));
-    test_generic_signer(dcke, drc, signer, message);
-
-    DNSKEYRecordContent pemDRC;
-    shared_ptr<DNSCryptoKeyEngine> pemKey{DNSCryptoKeyEngine::makeFromPEMString(pemDRC, signer.algorithm, signer.pem)};
-
-    BOOST_CHECK_EQUAL(pemKey->convertToISC(), dcke->convertToISC());
-
-    test_generic_signer(pemKey, pemDRC, signer, message);
-
-    auto dckePEMOutput = dcke->convertToPEMString();
-    BOOST_CHECK_EQUAL(dckePEMOutput, signer.pem);
-
-    auto pemKeyOutput = pemKey->convertToPEMString();
-    BOOST_CHECK_EQUAL(pemKeyOutput, signer.pem);
-  }
-}
-
-// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables,readability-identifier-length): Boost stuff.
 BOOST_AUTO_TEST_CASE(test_hash_qname_with_salt)
 {
   {
